docs/contributors_guide/Contribution_Procedure.rst

```diff
@@ -1,50 +1,108 @@
-     conventions where feasible
-   * Contributions should ideally include relevant spec and/or acceptance tests
+   * ``git clone git@github.com:<YOUR_GITHUB_NAME>/pupmod-simp-iptables iptables``
+
+#. Change into the repository's directory:
+
+   * ``cd iptables``
+
+#. Create a `feature branch`_: ``git checkout -b SIMP-XXXX``
+
+#. Do your work! `(Including tests, of course)`
+
+#. Commit your work. We will `squash`_ your `pull request`_ into one commit
+   when we merge it, so you can use as many commits as you'd like.
+
+   .. IMPORTANT::
+      The **first** commit should use the `Commit Message Conventions`_
+
+#. Push your changes to Github on your feature branch:
+
+   * ``git push origin SIMP-XXXX``
+
+#. Using the GitHub GUI, create a `pull request`_ from your feature branch to the branch of the
+   original repo that you want to contribute to.
+
+#. `Travis-CI`_ will run the spec tests for the branch and a member of the SIMP
+   team will review your submission. You should receive emails from Github as
+   the code reviews progress.
+
+#. If changes are requested, read the section below on amending your pull
+   request.
+
+Commit Message Conventions
+--------------------------
+
+An example commit message that following the SIMP conventions:
 
-#. Save your changes in a **single commit**
+  .. code-block:: none
 
-   * Use the following commit message conventions:
+     (SIMP-999) Fix the broken thing [50 chars max]
 
-     .. code-block:: none
+     Discussion about the fix (if needed) [each line: 72 chars max]
 
-        (SIMP-999) Fix the broken thing [50 chars max]
+     SIMP-998 #comment Comment on a related issue [72 chars max]
+     SIMP-999 #close
 
-        Discussion about the fix (if needed) [each line: 72 chars max]
+The first commit message should be the following format:
 
-        SIMP-998 #comment Comment on a related issue [72 chars max]
-        SIMP-999 #close
+  * First line:
 
-   * The commit message should be the following format:
+    * Start with the Issue name in parentheses [e.g., ``(SIMP-999)``], followed
+      by a summary of the change
+    * No longer than **50** characters
+    * Followed by a line of white space
 
-     * First line:
+  * Subsequent lines:
 
-       * Start with the Issue name in parentheses [e.g., ``SIMP-999``],
-         followed by a summary of the change
-       * No longer than **50** characters
-       * Followed by a line of white space
+    * Each line should be no longer than **72** characters
 
-     * Subsequent lines:
+  * Issue references:
 
-       * Each line should be no longer than **72** characters
+    * `JIRA issues can be referenced`_ at the end of the commit message
+    * It is recommended to only use `JIRA Smart Commit Tags`_ ``#comment`` and
+      ``#close``
+    * Avoid ``#resolve`` and ``#time`` as it will not update JIRA until
+      after the issue is merged
 
-     * Issue references:
+Contribution Procedure (Long Version)
```

Comment:
I agree with @lamawithonel about this―the initial section was originally intended to be a quickstart overview and would scan much more easily to new contributors if it were merely the steps' summaries + the commit message format, with links to the expanded procedure where needed.  Certainly, steps like `cd iptables` are overkill in an overview.

For a document intended to be rendered from reStructuredText and viewed online, I am not convinced an arbitrary limitation such as VT100 dimensions is appropriate, though.